### PR TITLE
tutorials-stateful-application-pr-2017-9-15-update

### PR DESCRIPTION
**Pull request by xiaosuiba**
update to 2017-9-15 version.

### CODE REVIEW

File: docs/tutorials/stateful-application/basic-stateful-set.md [hunk 1]
```diff
@@ -710,8 +712,11 @@ web-1   gcr.io/google_containers/nginx-slim:0.8
 web-2   gcr.io/google_containers/nginx-slim:0.8
 {% endraw %}```
 
-`web-0` has had its image updated, but `web-0` and `web-1` still have the original 
+<!--
+`web-0` has had its image updated, but `web-1` and `web-2` still have the original
```
Comment (zjj2wry) [inline docs/tutorials/stateful-application/basic-stateful-set.md:716]: has had, 感觉是上游写错了？是的话可以提个 pr 到上游修一下

File: docs/tutorials/stateful-application/basic-stateful-set.md [hunk 2]
```diff
@@ -27,7 +29,7 @@ following Kubernetes concepts.
 * [Cluster DNS](/docs/concepts/services-networking/dns-pod-service/)
 * [Headless Services](/docs/concepts/services-networking/service/#headless-services)
 * [PersistentVolumes](/docs/concepts/storage/volumes/)
-* [PersistentVolume Provisioning](http://releases.k8s.io/{{page.githubbranch}}/examples/persistent-volume-provisioning/)
```
Comment (zjj2wry) [inline docs/tutorials/stateful-application/basic-stateful-set.md:30]: 上游很多这种地址错误似乎 releases.k8s.io/，有时间可以帮忙fix一下

**Review by zjj2wry — APPROVED** [review]
generated lgtm

File: docs/tutorials/stateful-application/basic-stateful-set.md (hunk 1 above)
Comment (xiaosuiba) [reply docs/tutorials/stateful-application/basic-stateful-set.md:716]: 这里是表示 have its image updated 的过去完成时，have 变为 has had

Comment (zjj2wry) [reply docs/tutorials/stateful-application/basic-stateful-set.md:716]: ok, 知道啦

**Comment by zjj2wry** [comment]
@markthink

**Pull request merged**